core/src/avm1/globals/bitmap_data.rs

```diff
@@ -0,0 +1,1120 @@
+//! flash.display.BitmapData object
+
+use crate::avm1::activation::Activation;
+use crate::avm1::error::Error;
+use crate::avm1::function::{Executable, FunctionObject};
+use crate::avm1::object::bitmap_data::{BitmapDataObject, Color};
+use crate::avm1::{Object, TObject, Value};
+use crate::character::Character;
+use crate::display_object::TDisplayObject;
+use enumset::EnumSet;
+use gc_arena::MutationContext;
+use rand::Rng;
+
+pub fn constructor<'gc>(
+    activation: &mut Activation<'_, 'gc, '_>,
+    this: Object<'gc>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    let width = args
+        .get(0)
+        .unwrap_or(&Value::Number(0.0))
+        .coerce_to_i32(activation)?;
+
+    let height = args
+        .get(1)
+        .unwrap_or(&Value::Number(0.0))
+        .coerce_to_i32(activation)?;
+
+    if width > 2880 || height > 2880 || width <= 0 || height <= 0 {
+        log::warn!("Invalid BitmapData size {}x{}", width, height);
+        return Err(Error::ConstructorFailure);
+    }
+
+    let transparency = args
+        .get(2)
+        .unwrap_or(&Value::Bool(true))
+        .as_bool(activation.current_swf_version());
+
+    let fill_color = args
+        .get(3)
+        // can't write this in hex
+        // 0xFFFFFFFF as f64;
+        .unwrap_or(&Value::Number(4294967295_f64))
+        .coerce_to_i32(activation)?;
+
+    let bitmap_data = this.as_bitmap_data_object().unwrap();
+    bitmap_data.init_pixels(
+        activation.context.gc_context,
+        width as u32,
+        height as u32,
+        fill_color,
+    );
+    bitmap_data.set_transparency(activation.context.gc_context, transparency);
+
+    Ok(Value::Undefined)
+}
+
+pub fn get_height<'gc>(
```

Comment:
`get_height` => `height`

Let's go with the Rust naming convention of `height` for getters and `set_height` for setters. (I know we haven't been super consistent with this yet, but I'd like to start moving in that direction to keep a consistent style in the codebase).